libqtile/widget/backlight.py

```diff
@@ -84,7 +85,7 @@ def poll(self):
         return self.format.format(percent=percent)
 
     def change_backlight(self, value):
-        self.call_process(["xbacklight", "-set", str(value)])
+        self.call_process((self.change_command % value).split())
```

Comment:
Can you use shutil here instead?

Reply:
@tych0  why should we use different method as in other places? As I understand you speak about `shutil.run()`

Reply:
Oof, sorry. I meant shlex, not shutil. In particular, `shlex.split` instead of `.split()` so that we split things correctly if people use shell syntax in their configs.